test/e2e/lib/pages/editor-page.js

```diff
@@ -90,7 +92,8 @@ export default class EditorPage extends AsyncBaseContainer {
 		const fileNameInput = await driver.findElement( fileNameInputSelector );
 		await fileNameInput.sendKeys( file );
 		await driverHelper.elementIsNotPresent( driver, '.media-library__list-item.is-transient' );
-		await driverHelper.elementIsNotPresent( driver, '.media-library .notice.is-error' );
+		const errorShown = await this.isErrorDisplayed();
+		assert.strictEqual( errorShown, false, 'There is an error shown on the editor page!' );
```

Comment:
Maybe we can throw an error instead of using assert here?

Reply:
Done. Does [this](https://github.com/Automattic/wp-calypso/pull/33194/commits/4b01675d137dddebad0d8a461ee2d04fd5162d99) looks good?

Reply:
Not really. `isErrorDisplayed` returns a boolean in this case. e.g. test execution won't stop if the error notice is missing. in this case `assert` _will_ fail the test right away. Which is fine, as long as we fine with using asserts in non-spec files.

 ```suggestion
if ( errorShown ) {
		throw new Error( 'There is an error shown on the editor page!' );
}
```

This will throw an error that will should stop the test execution. And instead of the test failure, we should see a runtime error. This is not a big deal actually, so feel free to revert it back to using `assert`  if you will.

Reply:
Also done! :)